src/libcrun/container.c

```diff
@@ -1045,6 +1145,28 @@ libcrun_configure_libkrun (struct container_entrypoint_s *args, libcrun_error_t
 #endif
 }
 
```

Comment:
I think we would need `#if HAVE_WASMEDGE` for `libcrun_configure_wasmedge` as well as `#if HAVE_WASMER` for `libcrun_configure_wasmer`

Reply:
Merged `libcrun_configure_wasmedge` and `libcrun_configure_wasmer` into one function.

Reply:
@ibmibmibm makes sense. Could you also do a `clang-format` please.

Reply:
Tried `clang-format-10` and `clang-format-13`, and got different result. Which version should I use?

Reply:
I am using `clang-format version 12.0.1` but  anyways it should read `crun`'s format specs from `.clan-format` if you execute it from the `crun`'s base directory. `cd crun; clang-format -i src/libcrun/container.c` usually works for me.

Reply:
Done. no different to `clang-format version 13`.